linkerd/core/src/main/scala/io/buoyant/linkerd/FailureAccrualInitializer.scala

```diff
@@ -16,7 +16,7 @@ abstract class FailureAccrualConfig extends PolymorphicConfig {
   var backoff: Option[BackoffConfig] = None
 
   @JsonIgnore
-  val backoffOrDefault =
+  lazy val backoffOrDefault =
```

Comment:
Ideally the parser would spit out an immutable config, but, between jackson and inheritance, and polymorphism, that's not the world we live in.

Given that `backoff` has to be a var (sad face) I really think this should be a def.  Otherwise a stray early access of this field could lock it to an incorrect value.  In general, lazy vals in scala are kind of dangerous and hard to reason about so I like to avoid them except as a last resort.